projects/self-creation-portal/src/app/components/create-new/create-new.component.ts

```diff
@@ -0,0 +1,156 @@
+    { title: 'BROWSE_EXISTING', action: "", icon: 'search', url: 'browse-existing'},
+    { title: 'DRAFTS', action: "", icon: 'drafts', url: 'drafts' },
+    { title: 'SUBMITTED_FOR_REVIEW', action: "", icon: 'send', url: 'submit-for-review'},
+    { title: 'PUBLISHED', action: "", icon: 'published', url: 'published'},
+    { title: 'UP_FOR_REVIEW', action: "", icon: 'pending', url: 'up-for-review' }
+  ];
+
+  resourceList = [
+    { title: 'PROJECT', image:'./../assets/images/observation.svg', 
+      sidenav : [
+        { title: 'PROJECT_DETAILS', action: "", icon: 'add',  url: ''},
+        { title: 'TASKS', action: "", icon: 'search',  url: ''},
+        { title: 'SUBTASKS_AND_RESOURCES', action: "", icon: 'search',  url: ''},
+        { title: 'CERTIFICATE', action: "", icon: 'search',  url: ''},
+      ], showBackButton : true
+    },
+    { title: 'OBSERVATION', image:'./../assets/images/observation.svg', 
+      sidenav : [
+        { title: 'OBSERVATION_NAME'}
+      ], showBackButton : true 
+    },
+    { title: 'OBSERVATION_WITH_RUBRIC',image:'./../assets/images/observation.svg', 
+      sidenav : [
+        { title: 'OBSERVATION_NAME'}
+      ] 
+    },
+    { title: 'SURVEY', image: './../assets/images/survey.svg', 
+      sidenav : [
+        { title: 'SURVEY_NAME'}
+      ]
+    },
+    { title: 'PROGRAM',image: './../assets/images/survey.svg', 
+      sidenav : [
+        { title: 'PROGRAM_DETAILS'},
+        { title: 'RESOURCES'},
+        { title: 'RESOURCE_LEVEL_TARGETING'}
+      ] 
+    }
+  ];
+  
+  resourceHeader = {
+    "title":"PROJECT_NAME",
+    "buttons":[
+      { title: 'SAVE_AS_DRAFT'},
+      { title: 'PREVIEW'},
+      { title: 'SEND_FOR_REVIEW'}
+    ]
+  }
+
+  observationwithrubricsHeader = {
+    "title" : "OBSERVATION_FORM",
+    "buttons":[
+      { title: 'PAGINATION'},
+      { title: 'PROGRESS_STATUS'},
+      { title: 'SAVE_AS_DRAFT'},
+      { title: 'PREVIEW'},
+      { title: 'SEND_FOR_REVIEW'}
+    ]
+  }
+
+  constructor(private translate: TranslateService, private libsharedService:LibSharedModulesService, private dialog : MatDialog,) {
+    this.initializeTranslation();
+  }
+
+  private initializeTranslation(): void {
+    this.translate.setDefaultLang('en');
+  }
+
+  ngOnInit() {
+
+  }
+
+  onCardClick(cardItem: any) {
+    this.sidenavData = cardItem.sidenav
+    this.backButton = cardItem.showBackButton
+    this.subHeader = cardItem.subHeader;
+
+    if ((cardItem.title === 'PROJECT') || (cardItem.title === 'OBSERVATION') || (cardItem.title === 'SURVEY') || (cardItem.title === 'PROGRAM')) {
+      this.headerData =  this.resourceHeader;
+    }
+    else if (cardItem.title === 'OBSERVATION_WITH_RUBRIC') {
+      this.headerData = this.observationwithrubricsHeader;
+    }
+    else {
+        this.headerData = {};
+    }
+  }
+  
+  openPopup() {
+    const dialogConfig = new MatDialogConfig();
+    dialogConfig.backdropClass = 'cdk-overlay-dark-backdrop',
```

Comment:
can you try add this in dialog itself, rather than passing every time when we call dialog ?